examples/pqsc_helpers.py

```diff
@@ -0,0 +1,32 @@
+import zhinst.toolkit.driver.devices as devices
```

Comment:
can we consider to port this function into the toolkit PQSC driver? @tobiasah

Reply:
Or maybe even allowing direct indexing of instruments in the nodetree. So instead of writing something like 
`shfsg_zsync_port = pqsc_helpers.find_zsync_worker_port(pqsc, shfsg)
pqsc.zsyncs[shfsg_zsync_port].output.registerbank.enable(True)
pqsc.triggers.out[0].port(shfsg_zsync_port)`

we can write directly

`pqsc.zsyncs[shfsg].output.registerbank.enable(True)
pqsc.triggers.out[0].port(shfsg)`